bigbluebutton/greenlight
# CreateRoom permission for Sign-in + protecting backend

## Description

- Back-end endpoints are protected if user does not have permission
- User is redirected to `MainPage` on sign in if they don't have proper permission
## Testing Steps

- Turn `CreateRoom` permission to false for User role 
- Log in as regular user role
- Should be redirected to `/main-page` instead of `rooms`
## Screenshots (if appropriate):



## TODO
- Do the same for sign in
- UI
